physionet-django/user/forms.py

```diff
@@ -617,7 +617,8 @@ def clean(self):
         if state_required and not data['state_province']:
             raise forms.ValidationError("Please add your state or province.")
 
-        if not self.instance and CredentialApplication.objects.filter(user=self.user, status=0):
+        if not self.instance and CredentialApplication.objects\
```

Comment:
As above: There should be a space before the `\` I think, but PEP8 discourages `\` for wrapping long lines, suggesting parentheses instead: https://peps.python.org/pep-0008/#multiline-if-statements